### PR TITLE
create home page with navbar and footer component

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR request. The home page is created with navbar and footer components. The home page will be divided into smaller components in the next PR's. Please review and provide any suggestion.

### CODE REVIEW

File: src/components/footer/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import '../footer/footer.css';
+import person from '../../assets/images/person.png'
+
+export const Footer = () => {
+    return (
+        <>
```
Comment (JantuDeb) [inline src/components/footer/footer.jsx:6]: You can remove this

File: src/components/navbar/navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import '../navbar/navbar.css';
+import search from '../../assets/images/search.png';
+import Cart from '../../assets/images/cart.png';
+import wishlist from '../../assets/images/wishlist.png';
+import logout from '../../assets/images/logout.png';
+import { Link, NavLink } from 'react-router-dom';
+
+export const Navbar = () => {
+    return (
+        <>
```
Comment (JantuDeb) [inline src/components/navbar/navbar.jsx:10]: This can be removed

File: src/components/navbar/navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                            <div className="badge">
+                                <img src={wishlist} alt="wishlist image" className="badge__img" />
+                                <span className="badge__text">7</span>
+                            </div>
+                        </div>
+                    </NavLink>
+                    <NavLink to="/cart">
+                        <div className="cart-box">
+                            <div className="badge">
+                                <img src={Cart} alt="Cart image" className="badge__img" />
```
Comment (JantuDeb) [inline src/components/navbar/navbar.jsx:30]: Don't write image in alt prop, empty string is fine,

File: src/pages/homePage/home.jsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+import wishlist from '../../assets/images/wishlist.png';
+import { Footer } from "../../components/footer/footer";
+import { Link } from 'react-router-dom';
+
+
+export const Home = () => {
+    return (
+        <div className="home-container">
+            <Navbar />
+            <main className="main-container">
```
Comment (JantuDeb) [inline src/pages/homePage/home.jsx:14]: Divide this main into smaller components

**Review by JantuDeb — APPROVED** [review]
Added some comments

**Review by neetukumari4858 — APPROVED** [review]
overall look good just do some changes.

File: src/components/footer/footer.jsx (hunk 1 above)
Comment (ShreyashMahajan) [reply src/components/footer/footer.jsx:6]: ok

File: src/components/navbar/navbar.jsx (hunk 2 above)
Comment (ShreyashMahajan) [reply src/components/navbar/navbar.jsx:10]: ok

File: src/pages/homePage/home.jsx (hunk 4 above)
Comment (ShreyashMahajan) [reply src/pages/homePage/home.jsx:14]: yes, i will divide them in next pr's

File: src/components/navbar/navbar.jsx (hunk 3 above)
Comment (ShreyashMahajan) [reply src/components/navbar/navbar.jsx:30]: > overall look good just do some changes.

thank you i will do the changes

File: src/pages/homePage/home.jsx (hunk 4 above)
Comment (ShreyashMahajan) [reply src/pages/homePage/home.jsx:14]: thank you, i will do the changes.

**Pull request merged**